configurations/dockerDataTemplate.go

```diff
@@ -0,0 +1,124 @@
+package configurations
+
+import (
+	"fmt"
+	"strings"
+)
+
+type instruction interface {
+	WriteInstruction() string
+}
+
+type dockerfileData struct {
+	Stages []stage
```

Comment:
If we get rid of "stages" (as discussed earlier), then this struct is no longer required.

Reply:
Yes agreed